src/crypto/tls/common.go

```diff
@@ -465,6 +466,12 @@ const (
 	ECDSAWithSHA1 SignatureScheme = 0x0203
 )
 
+type TLSFlag uint16
```

Comment:
Let's make clear that this is the flag number (index) rather than the raw representation:
```suggestion
// TLSFlag represents the flag number.
type TLSFlag uint16
```